catalogue/webapp/pages/works.js

```diff
@@ -40,6 +40,7 @@ export const Works = ({
   if (works && works.type === 'Error') {
     return (
       <ErrorPage
+        title={works.httpStatus === 500 ? `We're experiencing technical difficulties at the moment. We're working to get this fixed by the end of the day.` : undefined}
```

Comment:
Let's maybe make the error message time-non-specific, as this might catch other errors too.